packages/react/src/user-info/UserInfo.tsx

```diff
@@ -238,13 +238,13 @@ function UserInfo(props: UserInfoProps) {
                             <a
                                 className="text-decoration-none text-muted align-self-center"
                                 href={
-                                    props.account.profile.deliveryServiceUrl +
+                                    props.account.profile.deliveryServices[0] +
```

Comment:
That won't work. The `deliveryServices[0]` is an ENS name. The app needs to resolve the ENS name and get the URL from the deliver service text record.